File: Python/Tarjeta.py
from payment import Payment

class Tarjeta(Payment):
    num_tarjeta = int
    cvv         = str
    fecha       = str

    def __init__(self, id, num_tarjeta, cvv, fecha):
        super(Tarjeta, self).__init__(id)
        self.num_tarjeta = num_tarjeta
        self.cvv         = cvv
        self.fecha       = fecha

File: Java/Tarjeta.java
package Java;

class Tarjeta extends Payment {
    Integer numero_tarjeta;
    String  cvv;
    String  fecha;

    public Tarjeta(Integer id, Integer numero_tarjeta, String cvv, String fecha){
        super(id);
        this.numero_tarjeta = numero_tarjeta;
        this.cvv = cvv;
        this.fecha = fecha;
    }

}
